OpenEdXMobile/src/main/java/org/edx/mobile/view/app_nav/CourseUnitNavigationActivity.kt

```diff
@@ -427,7 +427,11 @@ class CourseUnitNavigationActivity : BaseFragmentActivity(), CourseUnitFragment.
         val selectedComponent: CourseComponent? = if (resumedComponent != null) {
             resumedComponent
         } else {
-            subsection?.firstIncompleteComponent?.firstIncompleteComponent
+            if (isVideoMode) {
```

Comment:
can we update it as `subsection?.firstIncompleteComponent?.getFirstIncompleteComponent(isVideoMode)`

Reply:
`firstIncompleteComponent` written in java. so there is no default value of the parameter.